src/compiler/commandLineParser.ts

```diff
@@ -1347,11 +1339,11 @@ namespace ts {
                 const description = descriptionColumn[i];
                 result.push(optionName && `${tab}${tab}${optionName}${ description && (makePadding(marginLength - optionName.length + 2) + description)}`);
             }
-            if (configurations.files && configurations.files.length) {
+            if (fileNames.length) {

```

Comment:
The check used to be `fileNames && fileNames.length`, is this okay to abbreviate?